.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,31 @@
+name: shellcheck
+on:
+  pull_request:  
```

Comment:
maybe also on push? that way you can see something is wrong in your own branch before opening a PR

Reply:
> or you could just cheat and use some pre-made github action like https://github.com/marketplace/actions/shellcheck 😅

Does not work, only GitHub official or GardenLinux owned actions are allowed.. was my first try though :D